source/cmake/gui.cmake

```diff
@@ -117,13 +117,15 @@ endif()
 
 if(APPLE)
   
-  #Programmicon setzen
+
   set(MACOSX_BUNDLE_ICON_FILE muon.icns)
   set(myApp_ICON ${CMAKE_CURRENT_SOURCE_DIR}/gui/res/muon.icns)
   set_source_files_properties(${myApp_ICON} PROPERTIES MACOSX_PACKAGE_LOCATION "Resources")
-  #für OSX als Bundle bauen
+
   add_executable(muondetector-gui MACOSX_BUNDLE ${myApp_ICON} ${MUONDETECTOR_GUI_SOURCE_FILES} ${MUONDETECTOR_GUI_HEADER_FILES} ${MUONDETECTOR_GUI_UI_FILES} ${MUONDETECTOR_GUI_RESOURCE_FILES} ${qml_QRC} )  
 
+  set_target_properties(muondetector-gui PROPERTIES MACOSX_BUNDLE_INFO_PLIST ${CMAKE_SOURCE_DIR}/gui/res/Info.plist.in)
```

Comment:
It would be beneficial if the directory gui/res/ would be added as a CMake variable instead of writing it directly to the locations where it is needed.

Maybe "MUONDETECTOR_GUI_APPLE_RES_DIR" would be a good name.